firebase_test.go

```diff
@@ -337,6 +347,165 @@ func TestVersion(t *testing.T) {
 		}
 	}
 }
+func TestAutoInit(t *testing.T) {
+	tests := []struct {
+		name         string
+		confFilename string
+		initOptions  *Config
+		wantOptions  *Config
+	}{
+		{
+			"no environment var, no options",
```

Comment:
Please revise the names of these test cases. Some of them are outdated/invalid considering the changes we've done. Also lets use meaningful sentences like:

* No environment variable, and no explicit options
* Environment variable set to file, and no explicit options
* Environment variable set to string, and no explicit options

etc